src/tapis-api/ml-hub/models/inferenceAvailable.ts

```diff
@@ -0,0 +1,20 @@
+import { Models } from '@tapis/tapis-typescript';
+import { apiGenerator, errorDecoder } from 'tapis-api/utils';
+
+const downloadLinks = (
```

Comment:
Same as the comment above, this should be called something like `details`. The name of the function should be the same as the name of the file.